velox/expression/tests/ExpressionFuzzer.h

```diff
@@ -239,21 +239,6 @@ class ExpressionFuzzer {
 
   core::TypedExprPtr generateArg(const TypePtr& arg, bool isConstant);
 
-  /// Specialization for the "like" function: second and third (optional)
-  /// parameters always need to be constant.
-  std::vector<core::TypedExprPtr> generateLikeArgs(
```

Comment:
How are input parameters for these functions generated now? Or we just don't have these restrictions in the functions anymore?

Reply:
We have added const-ness to function signatures and Fuzzer uses that information to generate constant args.

```
std::vector<std::shared_ptr<exec::FunctionSignature>> likeSignatures() {
  // varchar, varchar -> boolean
  // varchar, varchar, varchar -> boolean
  return {
      exec::FunctionSignatureBuilder()
          .returnType("boolean")
          .argumentType("varchar")
          .constantArgumentType("varchar")
          .build(),
      exec::FunctionSignatureBuilder()
          .returnType("boolean")
          .argumentType("varchar")
          .constantArgumentType("varchar")
          .constantArgumentType("varchar")
          .build(),
  };
}

  registerFunction<
      Re2RegexpReplacePresto,
      Varchar,
      Varchar,
      Constant<Varchar>,
      Constant<Varchar>>({prefix + "regexp_replace"});
```